src/main/kotlin/org/rust/lang/core/psi/impl/mixin/RsArrayExprMixin.kt

```diff
@@ -0,0 +1,11 @@
+package org.rust.lang.core.psi.impl.mixin
+
+import com.intellij.lang.ASTNode
+import org.rust.lang.core.psi.RsArrayExpr
+import org.rust.lang.core.psi.RsExpr
+import org.rust.lang.core.psi.impl.RsCompositeElementImpl
+
+abstract class RsArrayExprMixin(node: ASTNode) : RsCompositeElementImpl(node), RsArrayExpr
```

Comment:
Given how the current code is organized now, I thought this was the right place for such extension method. `RustExprExtensions` doesn't look like a popular place among extensions at the time being :)

OK, I'll move the method there if it's the right place in the long term.